classpath.gradle

```diff
@@ -8,7 +8,7 @@ task classpath << {
     def listFiles = new File(exploded)
     if (listFiles.exists()) {
       listFiles.eachFileRecurse(){ file ->
-        if (file.name.endsWith("classes.jar")){
+        if (file.name.endsWith(".jar")){
```

Comment:
really can not understand why need change `classes.jar` to `.jar`,here line8 listFiles is in the buildDir?so the `.jar` is generated by gradle?